wasmedge.yaml

```diff
@@ -46,10 +47,14 @@ pipeline:
       patches: llvm-include.patch
 
   - runs: |
+      ln -s /usr/lib/llvm-19/lib/LLVMgold.so /usr/lib/ # I cannot convince ld to find this here
```

Comment:
@xnox Can you think of a reason I shouldn't symlink this in out clang package instead?

Reply:
Silly question it'll conflict which is why we moved it to begin with.